enginerpc/CMakeLists.txt.old

```diff
@@ -0,0 +1,148 @@
+##
```

Comment:
I think the commit of this file was not intended

Reply:
removed